.circleci/config.yml

```diff
@@ -67,15 +70,14 @@ jobs:
           name: Checkout next-ci-shared-helpers
           command: git clone --depth 1 git@github.com:Financial-Times/next-ci-shared-helpers.git .circleci/shared-helpers
       - *restore_npm_cache
+      - node/install-npm:
+          version: "7"
+      - run:
+          name: Install python
+          command: sudo apt-get update && sudo apt-get install python -qq
       - run:
           name: Install project dependencies
           command: make install
-      - run:
-          name: shared-helper / npm-install-peer-deps
-          command: .circleci/shared-helpers/helper-npm-install-peer-deps
-      - run:
```

Comment:
Question; it is correct to remove this job? (https://github.com/Financial-Times/session-decoder-js/blob/49bd53b8896b7cc5efd9c65a2ca651f348b25505/.circleci/config.yml#L78)

Reply:
Yep, this is a know problem in the npm migration, is documented here https://github.com/financial-times/next/wiki/npm-7-Migration-Guide#old-openssh-version-on-circleci and was apply like here https://github.com/Financial-Times/next-ammit-api/pull/796/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L74